crates/swc_css_parser/tests/recovery/at-rule/import/invalid-supports/output.swc-stderr

```diff
@@ -1,5 +1,5 @@
 
-  x Expected function or '(' token
```

Comment:
I think we should avoid using `token` in error messages, because developers can don't know what is `token`